src/page/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,133 @@
+import { useEffect, useState } from 'react';
+import { useParams, NavLink } from 'react-router-dom';
+import cn from 'classnames';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { getPeople } from '../../api';
+
+export const PeoplePage = () => {
+  const { slug } = useParams();
+  const [people, setPeople] = useState<Person[]>([]);
+  const [isLoading, setIsLoading] = useState<boolean>(false);
+  const [isEmpty, setIsEmpty] = useState<boolean>(false);
+  const [isDataFetched, setIsDataFetched] = useState<boolean>(false);
```

Comment:
You use this state variable to show the error message. But it's better to add `try catch` block to catch an error and set the proper value to smth like `isError` state variable.
Because now `isDataFetched` looks pretty similar to `isLoading`